src/main/kotlin/app/ui/AddRepoState.kt

```diff
@@ -60,6 +64,31 @@ class AddRepoState constructor(private val context: Context,
         Logger.info(Logger.Events.CONFIG_SETUP) { "Config setup" }
     }
 
+    private fun processPath(path: Path, hashContributorsForAllPaths: Boolean) {
```

Comment:
Remove hashContributorsForAllPaths and perform necessary changes mentioned before.

Reply:
removed